config.json

```diff
@@ -1998,6 +2007,22 @@
         "nugetVersion": "116.3.0",
         "nugetId": "Xamarin.Firebase.Analytics",
         "dependencyOnly": true
+      },
+      {
+        "groupId": "com.google.errorprone",
```

Comment:
I suspect these `dependencyOnly` entries won't be needed if the bound versions aren't reverted to earlier versions.

Reply:
There are 4 parallel PRs being worked on GPS, FB, MLKit and dependencies

Some of those could be from other PRs and I did not clean up `dependencyOnle = true`, but in this case this is false assumption:

```
Dependency errors : 3
1
	System.Exception: 
Artifact
	io.grpc.grpc-api:1.43.2
has unknown 'Runtime' dependency 
	com.google.errorprone.error_prone_annotations:2.9.0
Either fulfill or exclude this dependency.

2
	System.Exception: 
Artifact
	io.grpc.grpc-core:1.43.2
has unknown 'Runtime' dependency 
	com.google.errorprone.error_prone_annotations:2.9.0
Either fulfill or exclude this dependency.

3
	System.Exception: 
Artifact
	io.grpc.grpc-stub:1.43.2
has unknown 'Runtime' dependency 
	com.google.errorprone.error_prone_annotations:2.9.0
Either fulfill or exclude this dependency.
```